the-events-calendar.php

```diff
@@ -67,7 +67,7 @@ function tribe_events_not_php_version_plugin_name( $names ) {
 /**
  * Loads the action plugin
  */
-require_once dirname( TRIBE_EVENTS_FILE ) . '/src/Tribe/Main.php';
+require_once dirname(TRIBE_EVENTS_FILE) . '/src/Tribe/Main.php';
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 895d8cbff9dc608d7cca185dd40223b5</sub>